src/modules/esl-toggleable/core/esl-toggleable.shape.ts

```diff
@@ -19,6 +19,14 @@ export interface ESLToggleableTagShape<T extends ESLToggleable = ESLToggleable>
   /** Open toggleable marker. Can be used to define initial state */
   'open'?: boolean;
 
+  /**
+   * Define focus behaviour
+   *  - 'none' - no focus management
+   *  - 'chain' - focus on the first focusable element first and return focus to the activator after the last focusable element
+   *  - 'loop' - focus on the first focusable element and loop through the focusable elements
+   */
+  'focus-behaviour'?: 'none' | 'chain' | 'loop';
```

Comment:
be carefull in case you will replace behaviour by behavior as both options are possible

Reply:
by analogy with the properties:
 - [`scroll-behavior`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior)
 - [`transition-behavior`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-behavior)
 - etc

Reply:
I don't mind)